build/includes/sdk.mk

```diff
@@ -128,21 +128,23 @@ run-sdk-conformance-no-build: TIMEOUT ?= 30
 run-sdk-conformance-no-build: RANDOM := $(shell bash -c 'echo $$RANDOM')
 run-sdk-conformance-no-build: DELAY ?= $(shell bash -c "echo $$[ ($(RANDOM) % 5 ) + 1 ]")
 run-sdk-conformance-no-build: TESTS ?= ready,allocate,setlabel,setannotation,gameserver,health,shutdown,watch,reserve
+run-sdk-conformance-no-build: PORT ?= 59357
 run-sdk-conformance-no-build: ensure-agones-sdk-image
 run-sdk-conformance-no-build: ensure-build-sdk-image
-	DOCKER_RUN_ARGS="--network=host $(DOCKER_RUN_ARGS)" COMMAND=sdktest $(MAKE) run-sdk-command & \
-	docker run -p 59357:59357 -e "ADDRESS=" -e "TEST=$(TESTS)" -e "TIMEOUT=$(TIMEOUT)" -e "DELAY=$(DELAY)" \
-	 --net=host  $(sidecar_tag)
+	DOCKER_RUN_ARGS="--net host -e AGONES_SDK_GRPC_PORT=$(PORT) $(DOCKER_RUN_ARGS)" COMMAND=sdktest $(MAKE) run-sdk-command & \
```

Comment:
Right now all of the sdks with conformance tests use the grpc port, but the unity sdk (at least) uses the http port. I think @markmandel mentioned that you might have been working on some rest conformance tests as well, which would use the http port. Is there a way you are thinking that we could extend this to work with both the grpc and http ports?

Reply:
I have added a separate PR for Rest HTTP port testing:
 https://github.com/googleforgames/agones/pull/1079